frontend/src/css/cat-theme.css

```diff
@@ -198,3 +198,22 @@ Progess Steps
   min-width: 500px !important;
   max-width: 750px !important;
 }
+
+/* visually hidden, but stil accessible */
+.visually-hidden {
+  /* https://snook.ca/archives/html_and_css/hiding-content-for-accessibility */
+  position: absolute !important;
```

Comment:
how can you do this without `!important` and with inline CSS?

Reply:
dropped in 978baed; not needed.